.gitignore

```diff
@@ -77,3 +77,7 @@ packages/cli/.git-data.json
 dictionary.dic
 
 temp/
+
+# Docusaurus
+.docusaurus
+build/
```

Comment:
Lets move these up with the other docs ignores and prefix them with `docs/`.  Perhaps we also add a `/` to the end of .docusaurus.  Was looking for a dotfile until I ran the build